### ISSUE TITLE
melee attacks can be done if the player doesnt even face the target (aka behind)

### ISSUE THREAD

**Issue by budgebudge** [Atlantiss/NetherwingBugtracker#8169]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 
Melee attack in pvp can be done even when the character that hits someone turns into the opposite direction (if semi close enough).
so basically you can turn around to avoid gouge but still hit the target!!

**Current behaviour**:

**Expected behaviour**: 

**Server Revision**:

**Comment by Godmatik** [comment]
Intended - it's called facing leeway

#8061
